test/client/index.js

```diff
@@ -135,18 +135,6 @@ module.exports = config => {
             }
             return client.verifyTotpCode(client.totpAuthenticator.generate())
           })
-          .then(() => {
-            // The above enables TOTP on the account, but doesn't mark the
-            // session as being verified via TOTP, because it was already verified
-            // via email.  Create a new session that's explicitly TOTP-verified.
-            return client.setupCredentials(email, password)
-          })
-          .then(() => {
-            return client.auth(options)
-          })
-          .then(() => {
-            return client.verifyTotpCode(client.totpAuthenticator.generate())
-          })
```

Comment:
Just to check, this is no longer necessary because the "doesn't mark the session as being verified via TOTP" no longer applies, right?

Reply:
Sorry should have mentioned that, but you are correct. The session verificationMethod gets updated now.